src/k8s/pkg/snap/snap.go

```diff
@@ -354,4 +354,41 @@ func (s *snap) PreInitChecks(ctx context.Context, config types.ClusterConfig) er
 	return nil
 }
 
+func checkK8sServicePorts(config types.ClusterConfig, isControlPlane bool) error {
+	// NOTE(neoaggelos): in some environments the Kubernetes might hang when running for the first time
```

Comment:
The note should be left in it's original place

Reply:
Right, moved it by mistake. :)